CoinGeckoMarketCapAlgorithm.cs

```diff
@@ -37,7 +37,7 @@ public override void Initialize()
         {
             SetStartDate(2013, 10, 07);  //Set Start Date
             SetEndDate(2013, 10, 11);    //Set End Date
-            _equitySymbol = AddEquity("SPY").Symbol;
+            _equitySymbol = AddEquity("BTC").Symbol;
```

Comment:
`_cryptoSymbol = AddCrypto("BTCUSD").Symbol;`
replace the other `_equitySymbol` into `_cryptoSymbol`. Same for Python if any.

I'm not sure if the dates were having any data for the dataset, if not, please change it into later period 👍